setup.py

```diff
@@ -49,7 +49,7 @@
     "pytest==5.3.2",  # MIT license
     "pytest-cov==2.8.1",  # MIT
     "pytest-watch==4.2.0",  # MIT
-    "xain-sdk @ git+https://github.com/xainag/xain-sdk.git@development#egg=xain_sdk-0.2.0",
+    "xain-sdk @ git+https://github.com/xainag/xain-sdk.git@development#egg=xain_sdk-0.1.0",
```

Comment:
This seems weird, is it intentional?

Reply:
it is. we increased the egg info to 0.2.0 a few tickets before because we didn't know if it was needed due to caching reasons. actually it was not needed, therefor i reversed it with the current update to be consistent with the package version again.